packages/dev/test/RelayHub.test.ts

```diff
@@ -772,6 +775,56 @@ contract('RelayHub', function ([_, relayOwner, relayManager, relayWorker, sender
             expectEvent.inLogs(logs, 'TransactionRelayed', { status: RelayCallStatusCodes.PaymasterBalanceChanged })
           }
         })
+        context('with BatchGateway configured', function () {
+          const batchGateway = other
+
+          let gatewayForwarder: GatewayForwarderInstance
+          let relayHubInstance: RelayHubInstance
+          let recipientContract: TestRecipientInstance
+          let relayRequest: RelayRequest
+
+          beforeEach(async function () {
+            relayRequest = cloneRelayRequest(sharedRelayRequestData)
+            gatewayForwarder = await GatewayForwarder.new()
+            relayHubInstance = await deployHub(stakeManager.address, penalizer.address, batchGateway)
+            recipientContract = await TestRecipient.new(gatewayForwarder.address)
+            await gatewayForwarder.setTrustedRelayHub(relayHubInstance.address)
+            await paymasterContract.setTrustedForwarder(gatewayForwarder.address)
+            await paymasterContract.setRelayHub(relayHub)
+            await relayHubInstance.depositFor(paymasterContract.address, {
+              from: senderAddress,
+              value: ether('1')
+            })
+            relayRequest.request.to = recipientContract.address
+            relayRequest.request.data = recipientContract.contract.methods.emitMessageNoParams().encodeABI()
+            relayRequest.relayData.paymaster = paymasterContract.address
+            relayRequest.relayData.forwarder = gatewayForwarder.address
+          })
+
+          it('should reject relayCall with non-empty signature coming from the BatchGateway', async function () {
```

Comment:
wrong test: 
should reject empty signature coming from non-batch-gateway (also commented on the relayhub on this)

Reply:
Fixed.